--- git-store-meta.pl	2020-11-11 15:59:33.000000000 -0500
+++ new.pl	2020-11-11 16:00:59.000000000 -0500
@@ -48,6 +48,7 @@
 #
 # git-store-meta 2.0.1
 # Copyright (c) 2015-2019, Danny Lin
+# Custom DST version for git-logic-init by Krishna Bhamidipati
 # Released under MIT License
 # Project home: https://github.com/danny0838/git-store-meta
 # =============================================================================
@@ -539,7 +540,7 @@
     my ($dev, $ino, $mode, $nlink, $uid, $gid, $rdev, $size, $atime, $mtime, $ctime, $blksize, $blocks) = lstat($file);
     my ($user) = getpwuid($uid);
     my ($group) = getgrgid($gid);
-    $mtime = timestamp_to_gmtime($mtime);
+    $mtime = timestamp_to_gmtime_dst($mtime);
     $atime = timestamp_to_gmtime($atime);
     $mode = sprintf("%04o", $mode & 07777);
     $mode = "0664" if $type eq "l";  # symbolic do not apply mode, but use 0664 if checked out as a plain file
@@ -883,7 +884,7 @@
                 warn "warn: `$File' cannot set acl to '$data{'acl'}'\n" if !$check;
             }
             if ($fields_used{'mtime'} && $data{'mtime'} ne "") {
-                my $mtime = gmtime_to_timestamp($data{'mtime'});
+                my $mtime = gmtime_to_timestamp_dst($data{'mtime'});
                 my $atime = (lstat($file))[8];
                 print "`$File' set mtime to '$data{'mtime'}'\n" if $argv{'verbose'};
                 if (!$argv{'dry-run'}) {
